test/v2/ThirdPartyRegistry.spec.js

```diff
@@ -2404,7 +2398,7 @@ describe('ThirdPartyRegistry', function () {
       expect(thirdParty.metadata).to.be.eql(thirdParty1[1])
       expect(thirdParty.resolver).to.be.eql(thirdParty1[2])
       expect(thirdParty.isApproved).to.be.eql(initialValueForThirdParties)
-      expect(thirdParty.maxItems).to.be.eq.BN(TIERS[0].value)
+      expect(thirdParty.maxItems).to.be.eq.BN(10)
```

Comment:
What do you think of creating a variable with this value like `itemsToBuy` or something

Reply:
Created the `slotsToAddOrBuy` variable used for all these cases, (10 is the most prevalent one)